src/patternfly/components/Nav/nav.scss

```diff
@@ -3,6 +3,29 @@
   --pf-c-nav--Transition: var(--pf-global--Transition);
   --pf-c-nav__item--m-expanded__toggle-icon--Transform: rotate(90deg);
 
+  // dark modifier
+  --pf-c-nav--m-dark__list-link--Color: var(--pf-global--Color--light-300);
+  --pf-c-nav--m-dark__list-link--m-current--Color: var(--pf-global--Color--light-100);
+  --pf-c-nav--m-dark__list-link--hover--Color: var(--pf-global--Color--light-100);
+  --pf-c-nav--m-dark__list-link--focus--Color: var(--pf-global--Color--light-100);
+  --pf-c-nav--m-dark__list-link--active--Color: var(--pf-global--Color--light-100);
+  --pf-c-nav--m-dark__simple-list-link--Color: var(--pf-global--Color--light-300);
+  --pf-c-nav--m-dark__simple-list-link--hover--Color: var(--pf-global--Color--light-100);
+  --pf-c-nav--m-dark__simple-list-link--focus--Color: var(--pf-global--Color--light-100);
+  --pf-c-nav--m-dark__simple-list-link--active--Color: var(--pf-global--Color--light-100);
+  --pf-c-nav--m-dark__simple-list-link--hover--BackgroundColor: var(--pf-global--BackgroundColor--dark-400);
+  --pf-c-nav--m-dark__simple-list-link--focus--BackgroundColor: var(--pf-global--BackgroundColor--dark-400);
+  --pf-c-nav--m-dark__simple-list-link--active--BackgroundColor: var(--pf-global--BackgroundColor--dark-400);
+  --pf-c-nav--m-dark__simple-list-link--m-current--Color: var(--pf-global--Color--light-100);
+  --pf-c-nav--m-dark__simple-list-link--m-current--FontWeight: var(--pf-global--FontWeight--normal);
+  --pf-c-nav--m-dark__simple-list-link--m-current--BackgroundColor: var(--pf-global--BackgroundColor--dark-400);
+  --pf-c-nav--m-dark__item--m-expanded__simple-list-link--Color: var(--pf-global--Color--light-100);
```

Comment:
Should this be `--pf-c-nav--m-dark__item--m-current-list-link--Color` instead? same for the var below?